src/Finder/TransformedFinder.php

```diff
@@ -80,6 +80,16 @@ public function findPaginated($query, $options = [])
         return new Pagerfanta(new FantaPaginatorAdapter($paginatorAdapter));
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function findHybridPaginated($query)
+    {
+        $paginatorAdapter = $this->createHybridPaginatorAdapter($query);
```

Comment:
don’t we need some code here to ensure we have a query instance?

Reply:
Hi @lsmith77  , it's done in `createHybridPaginatorAdapter` -> `$query = Query::create($query);` -> `throw new InvalidException('Unexpected argument to create a query for.');`

Reply:
ah thx.

Reply:
> I guess this is convenient syntax sugar?

Yes. It avoid me to add this function at two places in a project : 
https://www.strangebuzz.com/en/blog/implementing-a-search-engine-with-elasticsearch-and-symfony-part-2 
Check out the commented line.

> shouldn’t we then also one such method using createRawPaginatorAdapter()?

I am not sure to understand what you mean.

Reply:
ie. adding a `findRawPaginated()`

Reply:
> ah thx.

The `$queryObject = Query::create($query);` call in the `findPaginated` function seems useless as it's already done in `createPaginatorAdapter` function.